02_knight_and_princess/tamara_gadieva/graph_func.cpp

```diff
@@ -0,0 +1,43 @@
+#include "graph_generate.hpp"
+
+void Graph::add_vertex() {
+  vertices_.push_back(Vertex(get_new_vertex_id()));
+}
+
+void Graph::add_vertex_inf(const VertexId& cur_vertex_id) {
+  vertices_[cur_vertex_id].add_edge_id(get_cur_edge_id());
+}
+
+void Graph::add_edge(const VertexId& from_vertex_id,
+                     const VertexId to_vertex_id) {
+  edges_.push_back(Edge(get_cur_edge_id(), from_vertex_id, to_vertex_id));
+  increase_edge_id();
+}
+
+std::string Graph::json_string() const {
+  std::string result_graph;
```

Comment:
Это ведь не `graph` а `string` или просто `result`...
```suggestion
  std::string result;
```